INSTALL.txt

```diff
@@ -0,0 +1,27 @@
+Installation instructions
+=========================
+
+Minimal tools needed
```

Comment:
You might want to put up front that these instructions are for RHEL/CentOS/SL 7 and will need some modifications to work on Debian, for example. (We ought to write instructions for Debian and other distros eventually but one thing at a time I guess.)